Automattic/jetpack
# Contact Form: refactor editor to use forms package

## Proposed changes:

As part of the Contact Form module deprecation, this PR refactors the `Grunion_Editor_View` class to reference the Forms package.

The refactored code calls the static methods of `Editor_View`.

### Other information:


## Jetpack product discussion


Project thread: pf5801-Aj-p2
Issue: https://github.com/Automattic/jetpack/issues/35653

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






Let's do some regression testing.

- Spin up a test site with this branch
- Update the file `/projects/plugins/jetpack/modules/contact-form.php` so the condition on line 33 is falsy (or comment the block). This ensures we're loading the Contact Form module.
- Create a new post
- Check you can still add a Contact Form block and that nothing breaks
